client/landing/gutenboarding/stores/onboard/actions.ts

```diff
@@ -105,6 +107,7 @@ export function* createSite( {
 			...( anchorFmSpotifyUrl && {
 				anchor_fm_spotify_url: anchorFmSpotifyUrl,
 			} ),
+			is_blank_canvas: isBlankCanvasDesign( selectedDesign ),
```

Comment:
Are we sure that the `selectedDesign` is always defined at this point? I'm asking this since we're using optional chaining for it at [several](https://github.com/Automattic/wp-calypso/blob/b6b2d670bb7de42499fe437bb9b6f45a514ffd33/client/landing/gutenboarding/stores/onboard/actions.ts#L67) [places](https://github.com/Automattic/wp-calypso/blob/b6b2d670bb7de42499fe437bb9b6f45a514ffd33/client/landing/gutenboarding/stores/onboard/actions.ts#L91) in the code and `isBlankCanvasDesign` doesn't accept `undefined`.

Reply:
Good catch. Fixed in https://github.com/Automattic/wp-calypso/pull/52288/commits/8bd7634f19438bf7cc5769df1fb2cd905ef2a991